frontend/src/components/core/Sidebar/ThemedSidebar.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from "react"
+import PageLayoutContext from "components/core/PageLayoutContext"
+import ThemeProvider from "components/core/ThemeProvider"
+import { createTheme, ThemeConfig } from "theme"
+import Sidebar, { SidebarProps } from "./Sidebar"
+
+const createSidebarTheme = (theme: ThemeConfig): ThemeConfig =>
+  createTheme(
+    {
+      sidebar: theme.emotion.colors.bgColor,
+      main: theme.emotion.colors.secondaryBg,
+      name: "Sidebar",
+    },
+    theme
+  )
+
+const ThemedSidebar = ({
+  theme,
```

Comment:
nit: `theme` doesn't seem to be used below (although it will be if things can be rewritten to use `useTheme`)

Reply:
It was included in order to pull it out of props. This way we can just pass props down to Sidebar and it would only have sidebar props. Renamed to SidebarProps